### PR TITLE
file structre and setup for client and server sides #15

### PR DESCRIPTION
**Pull request by Hadeel-Salamin**

### CODE REVIEW

File: client/src/logo.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (shahenazmonia) [inline client/src/logo.svg]: Delete unrequired files

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "reservationsystem",
+  "version": "1.0.0",
+  "description": "A web app that enables the YDRC staff to coordinate the reservation process in their organization between all of them.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "client": "cd client && npm start",
+    "start": "nodemon start ./server/server.js && npm run client",
```
Comment (shahenazmonia) [inline package.json]: You have to put ```nodemon``` inside ```dev``` script or anything else except ```start``` script. You can't  run the code on heroku with ```nodemon``` module

File: package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/FACK1/ReservationSystem/issues"
+  },
+  "homepage": "https://github.com/FACK1/ReservationSystem#readme",
+  "dependencies": {
+    "express": "^4.16.4",
+    "nodemon": "^1.18.10"
```
Comment (shahenazmonia) [inline package.json]: ```nodemon``` should be in devDependencies

File: client/src/App.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
```
Comment (shahenazmonia) [inline client/src/App.js]: You don't need this image and this style in your app, add any default ```div``` at the beginning

File: package.json (hunk 3 above)
Comment (Hadeel-Salamin) [reply package.json]: done

File: client/src/logo.svg (hunk 1 above)
Comment (Hadeel-Salamin) [reply client/src/logo.svg]: done

File: package.json (hunk 2 above)
Comment (Hadeel-Salamin) [reply package.json]: done

**Pull request merged**